client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -569,7 +582,9 @@ export class CheckoutThankYouHeader extends PureComponent {
 						) : (
 							<h2 className="checkout-thank-you__header-text">{ this.getText() }</h2>
 						) }
-
+						{ primaryPurchase && selectedSite?.ID && isTitanMail( primaryPurchase ) && (
```

Comment:
G Suite suffers from the same issue. Shouldn't we also add it here?

Reply:
Do you mind if I take that on in a follow-up PR?
I think we may need to consider the overall UX for G Suite as well, and I'd rather land this and then iterate (as the behaviour for G Suite has been there for longer!)

Reply:
No I don't.